runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -275,13 +278,29 @@ else if (client.getWidget(WidgetInfo.QUEST_COMPLETED_NAME_TEXT) != null)
 			String text = client.getWidget(WidgetInfo.QUEST_COMPLETED_NAME_TEXT).getText();
 			fileName = "Quest(" + text.substring(19, text.length() - 1) + ")";
 		}
+		else if (hasBeenKilled)
+		{
+			hasBeenKilled = false;
+			takeScreenshot(client.getLocalPlayer().getName() + " Died on " + LocalDate.now());
```

Comment:
This is already stored in a player-specific directory, don't add playername again in the filename.